plugins/com.google.cloud.tools.eclipse.appengine.login/src/com/google/cloud/tools/eclipse/appengine/login/GoogleLoginService.java

```diff
@@ -127,6 +127,14 @@ Credential createCredentialHelper(String accessToken, String refreshToken) {
     return credential;
   }
 
+  public void clearCredential() {
+    MApplication application = PlatformUI.getWorkbench().getService(MApplication.class);
+
+    Map<String, Object> synchronizedMap =
```

Comment:
> other callers to this method will get a new synchronizedMap.

Ah, I see. I may have misunderstood how sychronizedMap can work. Then I should re-work this thing in the other methods too. (Not in this PR.) I think for now I'll remove the synchronizedMap here, and do something about the re-work in another PR.